server/bundler/plugin.js

```diff
@@ -30,11 +31,12 @@ ChunkFileNames.prototype.apply = function( compiler ) {
 						"script.onerror = script.onload = script.onreadystatechange = null;",
 						"delete installedChunks[ chunkId ];",
 						"window.__chunkErrors[ " + JSON.stringify( chunkMaps.name ) + "[chunkId]||chunkId ]=new Error();",
-						"callback.call( null, " + this.requireFn + ")"
+						"return Promise().resolve();"
```

Comment:
> Or see if we can ditch our custom loader altogether.

You mean this plugin, or https://github.com/Automattic/wp-calypso/blob/master/server/bundler/loader.js ?

We might be able to ditch this plugin, if we can look for errors in the returned rejected promise from `require.ensure` in `bundler/loader` — which handles retry logic atm using the custom global `__chunkErrors`.

Another difference seems to be https://github.com/Automattic/wp-calypso/pull/10799/files#diff-af0ff8f84f0c14d14267bd8ccd37db5fR41 — where we're serving minified assets, depending on debug environment. We could likely just minify in webpack instead.